oadp-operator/controllers/registry_test.go

```diff
@@ -0,0 +1,412 @@
+package controllers
+
+import (
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"reflect"
+	"testing"
+)
+
+func TestVeleroReconciler_buildRegistryDeployment(t *testing.T) {
+	tests := []struct {
+		name               string
+		registryDeployment *appsv1.Deployment
+		bsl                *velerov1.BackupStorageLocation
+		wantErr            bool
```

Comment:
do you even plan to test the error? since it is returned from controller-runtime, you can ignore it and remove this field from the struct totally.

Reply:
May be we want this in future tests, but yeah I would say remove it if it's not being used

Reply:
Removed unused yest args and updated them for all the unit tests.